File: examples/TeamUpdateExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
    # access_token="YOUR_ACCESS_TOKEN",
)

with ApiClient(configuration) as api_client:
    team_update_request = models.TeamUpdateRequest(
        name="New Team Name",
    )

    try:
        response = api.TeamApi(api_client).team_update(
            team_update_request=team_update_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling TeamApi#team_update: %s\n" % e)

File: sandbox/node/src/TeamUpdateExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TeamApi();
apiCaller.username = "YOUR_API_KEY";
// apiCaller.accessToken = "YOUR_ACCESS_TOKEN";

const teamUpdateRequest: models.TeamUpdateRequest = {
  name: "New Team Name",
};

apiCaller.teamUpdate(
  teamUpdateRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling TeamApi#teamUpdate:");
  console.log(error.body);
});
